contribute/style-guides/frontend.md

```diff
@@ -5,6 +5,7 @@ Generally we follow the Airbnb [React Style Guide](https://github.com/airbnb/jav
 ## Table of Contents
 
 - [Frontend Style Guide](#frontend-style-guide)
+
```

Comment:
Nit. You should definitely investigate why your local setup makes these changes.